README.md

```diff
@@ -118,6 +118,9 @@ When creating an instance of SubtitleOctopus, you can set the following options:
   occurs if browser doesn't support web workers). (Optional)
 - `debug`: Whether performance info is printed in the console. (Default:
   `false`)
+- `prescaleTradeoff`: Scale down (`< 1.0`) the subtitles canvas to `softHeightLimit` to improve speed, or scale it up (`> 1.0`) to improve quality. (Default: `null` - no scaling)
```

Comment:
https://github.com/jellyfin/JavascriptSubtitlesOctopus/pull/11#discussion_r398042544
_GitHub doesn't open resolved discussions for some reason_
> added ability to pass in null meaning "disable scaling" (also defaulted to it)

If `prescaleTradeoff === 1.0`, we are just limited to a range of `[softHeightLimit, hardHeightLimit]` and no scaling is applied.

Reply:
Limit-width floating-point's inherent inaccuracies often can lead to unforseen errors; however even 32bit IEEE 754 floats can accurately represent **integers** between 0 and 16777216. Therefore multiplying with 1.0 should not introduce any discrepancies.

Now in case of the simplified logic suggested below, there is a multiplication by `newH / height`. If `prescaleFactor` was 1.0 and the original size was integers, there's still no issue; however when the original size was not an integer due to the `pixelAspectRatio`, there can potetntially be small errors. If this is of concern, there can still be a fast path added like before for `prescaleTradeoff === null`, but instead for `prescaleFactor == 1.0` which is more intuitive.

Reply:
Perhaps JustAMan expected that `null` as "disable" would be clearer than `1`.

Reply:
Note that since `(null <= 0) === true`, `null`will still act as a noop with the fixed and simplified logic even if it's not the default.